README.md

```diff
@@ -149,7 +149,7 @@ coming soon
 
 ### Deployment
 
-coming soon
+https://ankitt26.github.io/Portfolio-setup/
```

Comment:
- [ ] It would be better to put the link in the "Live Demo" section because at the moment it has the text "coming soon" written there which is not correct.

Reply:
solved